File: app/src/main/java/com/example/lunch_app/Hotpepperapi.java
package com.example.lunch_app;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.text.Html;
import android.text.method.LinkMovementMethod;
import android.text.method.MovementMethod;
import android.widget.ArrayAdapter;
import android.widget.ListView;
import android.widget.TextView;

import com.example.lunch_app.model.Gourmet;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface Hotpepperapi {

    @GET("/hotpepper/gourmet/v1/")
    Call<Gourmet> webservice(
            @Query("key") String key,
            @Query("lat") String lat,
            @Query("lng") String lng,
            @Query("range") String range,
            @Query("lunch") String lunch,
            @Query("count") String count,
            @Query("format") String format
//            @Query("name") String name,
//            @Query("genre") String genre,
//            @Query("free_food") String free_food,
//            @Query("private_room") String private_room,
//            @Query("title") String title
    );
}

File: app/src/main/java/jp/co/pannacotta/lunch_app/Hotpepperapi.kt
package jp.co.pannacotta.lunch_app

import jp.co.pannacotta.lunch_app.model.Gourmet
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface Hotpepperapi {
    @GET("/hotpepper/gourmet/v1/")
    fun webservice(
            @Query("key") key: String?,
            @Query("lat") lat: String?,
            @Query("lng") lng: String?,
            @Query("range") range: String?,
            @Query("lunch") lunch: String?,
            @Query("count") count: String?,
            @Query("format") format: String?
            //            @Query("name") String name,
            //            @Query("genre") String genre,
            //            @Query("free_food") String free_food,
            //            @Query("private_room") String private_room,
            //            @Query("title") String title
    ): Call<Gourmet?>?
}